tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -0,0 +1,41 @@
+package e2e_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/oadp-operator/tests/e2e/lib"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+var _ = Describe("VM backup and restore tests", Ordered, func() {
+	var v *VirtOperator
+	var err error
+
+	var _ = BeforeAll(func() {
+		if !virtTestingEnabled {
+			Skip("Virtualization testing is disabled, skipping tests")
+		}
+
+		Expect(runTimeClientForSuiteRun).ToNot(BeNil())
```

Comment:
in `e2e_suite_test.go`we check if no error occurred when creating each client, so I think we do need to check if they are not nil here

Reply:
Was this "do not need to check"? I took these checks out, but I can add checks for every client instead if I read this the wrong way.

Reply:
yes yes, forget to add "not", thanks!